apps/web/vibes/soul/examples/sections/cart/electric.tsx

```diff
@@ -158,7 +160,8 @@ async function getCart() {
         JSON.stringify(generateCartFromLineItems(defaultLineItems)),
     ),
   );
-  return cart;
+
+  return { ...cart, couponCodeAction };
```

Comment:
This is meant to provide cart data, not props. We should instead pass the `couponCodeAction` prop directly in JSX